test/msw-testing-lib-react/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@vitest/test-testing-lib-react-msw",
+  "private": true,
+  "scripts": {
+    "build": "tsc && vite build",
+    "coverage": "vitest --coverage",
+    "dev": "vite",
+    "preview": "vite preview",
+    "test": "vitest"
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
+    "@vitejs/plugin-react": "^1.1.3",
+    "happy-dom": "*",
```

Comment:
I believe you're using jsdom. Not happy-dom (I think happy-dom also gives a react warning if it was used because render from testing lib already does some cleanup).

Reply:
oooo :clap: :clap: